Session6/Polynomial_Operations/PolyTest.java

```diff
@@ -0,0 +1,93 @@
+package Polynomial_Operations;
+
+import static org.junit.Assert.*;
+
+import java.lang.Character.Subset;
+import java.util.Arrays;
+import java.util.Collection;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+
+@RunWith(Parameterized.class)
+public class PolyTest {
+	
+	public Poly Poly =new Poly();
+
+	private int input;
+	private int expectedValue;
+	private float value ;
+	private float evaluatedValue;
+	private String valueAdd;
+	private String evaluatedValueAdd;
+	private String valueMultiply;
+	private String evaluatedValueMultiply;
+	private Poly add1 =new Poly();
+	private Poly add2 =new Poly();
+	private Poly mul1 =new Poly();
+	private Poly mul2 =new Poly();
+	
+	
+	public PolyTest(int expectedValue, int input, float value ,float evaluatedValue ,
+			Poly add1,Poly add2 , String evaluatedValueAdd,Poly mul1,Poly mul2, String evaluatedValueMultiply ) {
+
+		// is test of degree
+		this.input = input;
+		this.expectedValue = expectedValue;
+
+		// is test of evaluation of polynomial 
+		this.value = value;
+		this.evaluatedValue = evaluatedValue;
+
+		// is test of Add of polynomial
+		this.evaluatedValueAdd = evaluatedValueAdd;
+		this.add1 = add1 ;
+		this.add2 = add1;
+		
+		// is test of multiply of Polynomial
+		this.evaluatedValueMultiply = evaluatedValueMultiply;
+		this.mul1 =mul1;
+		this.mul2 =mul2;
+	}
+
+	@Parameterized.Parameters
+	public static Collection data() {
+		return Arrays.asList(new Object[][] { 
+			{  new Poly(new int[] { 6, 5, 2 }).degree(), 2,

```

Comment:
Here we are only supposed to pass values directly. We don't call the function over here